### PR TITLE
Setup vagrant with conan

### PR DESCRIPTION
**Pull request by hklabbers**

### CODE REVIEW

File: etc/install_on_vagrant.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+conan profile show clang-6.0-linux-x86_64
+conan remote list
+
+#Install python dependencies
+pip3 install pystache antlr4-python3-runtime
+pip3 install jsonschema psutil filemagic
+
+#Build includeos
+cd ../..
+conan create IncludeOS includeos/test -pr clang-6.0-linux-x86_64
```
[the remark this answers is not in the document]
Comment (hklabbers) [reply etc/install_on_vagrant.sh]: You are right. It should come from conan repo. The idea is to use Vagrant to contribute to IncludeOS development. So I'm changing the install_on_vagrant to reflect that.

File: Vagrantfile [hunk 2]
```diff
@@ -5,7 +5,8 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.define "IncludeOS" do |config|
-    config.vm.box = "ubuntu/xenial64"
+    config.vm.box = "hklabbers/Ubuntu-18.04-Dev-40GB"
```
[the remark this answers is not in the document]
Comment (hklabbers) [reply Vagrantfile]: Actually that is not. The ubuntu/xenial64 has a 20GB harddisk, this was/is not enough for compiling IncludeOS. I will check whether 20GB is enough for IncludeOS with current setup.

File: README.md [hunk 3]
```diff
@@ -111,6 +111,93 @@ We upload to two channels:
...
+```
+> **NOTE:** We only guarantee that the **latest 10 packages** are kept in the
+`latest` channel. All `stable` packages will be kept in the stable channel unless
+proven unsafe. One suggested workaround is to copy packages into your own repository.
+YOu can do that using the copy command:
+
+```
+    $ conan copy --all -p <path-to-conanfile> <package_name>@<your_repository>/<your_channel>
+```
+___
```
Comment (mnordsletten) [inline README.md]: The part above of the README has changed on the dev branch. I think you have merged in some old changes.

File: README.md [hunk 4]
```diff
@@ -111,6 +111,93 @@ We upload to two channels:
...
+```
+##### Now you can create the Vagrant box with IncludeOS for development of services and login to the Vagrant box:
+```
+	$ vagrant up
+	$ vagrant ssh
+```
+Test the vagrant box installation by creating your first IncludeOS service. Inside Vagrant IncludeOS has been built with profile 'clang-6.0-linux-x86_64', use this profile to create the demo service. See the following paragraph.
+___
+
+### <a name="building_service"></a>Building, starting and creating your first IncludeOS Service with Conan
```
Comment (mnordsletten) [inline README.md]: All below this point is also old and outdated.

**Pull request merged**